browser/farbling/BUILD.gn

```diff
@@ -15,6 +15,7 @@ if (!is_android) {
       "brave_navigator_devicememory_farbling_browsertest.cc",
       "brave_navigator_hardwareconcurrency_farbling_browsertest.cc",
       "brave_navigator_keyboard_api_browsertest.cc",
+      "brave_navigator_languages_farbling_browsertest.cc",
```

Comment:
why is this in farbling when the other parts of the code are in brave shields?

Reply:
I guess more generally why does this directory exist if it's only tests? That seems a bit odd to me